feat: :sparkles: adding the navbar to the project

colmedev opened the pull request:
adding the navbar to the project but the burger menu isn't work and the navbar in desktop still fixed 

![image](https://github.com/AnaRangel/anarangel.github.io/assets/161905693/971a2b9d-99e7-40b9-8305-c0db7beaf724)

![image](https://github.com/AnaRangel/anarangel.github.io/assets/161905693/5a925143-0840-4bf3-8358-4c0677f09787)

---
UPDATE
The burger menu works!! 



https://github.com/AnaRangel/anarangel.github.io/assets/161905693/bce870f7-132a-486c-bd4b-b5d54b0fd937

Others
Fix the html semantic

---
UPDATE
The navbar hide in desk when scrolls down and appear when scrolls up and add menu burger animations

https://github.com/AnaRangel/anarangel.github.io/assets/161905693/812917db-3c42-42a6-b359-1a39545a790b


https://github.com/AnaRangel/anarangel.github.io/assets/161905693/000a3975-a41b-48fa-814c-8f007fac4e48


---
Size Update (zyruks help me)

![image](https://github.com/AnaRangel/anarangel.github.io/assets/161905693/913af2fe-b31e-4ecb-bc6f-c15eeca375e1)

![image](https://github.com/AnaRangel/anarangel.github.io/assets/161905693/afff8009-862c-4b0b-ab5f-61627f9959a4)

colmedev committed:
feat: :sparkles: adding the navbar to the project

adding the navbar to the project but the burger menú isn't work

colmedev committed:
feat: :sparkles: adding the navbar to the project

adding the navbar to the project but the burger menú isn't work

colmedev committed:
feat: :sparkles: adding the navbar to the project

adding the navbar to the project but the burger menú isn't work

colmedev committed:
feat: :sparkles: adding the navbar to the project

adding the navbar to the project but the burger menú isn't work

colmedev committed:
feat: :sparkles: adding the navbar to the project

adding the navbar to the project but the burger menú isn't work

Zyruks commented on src/components/Navbar/Navbar.astro:
```diff
@@ -0,0 +1,142 @@
+---
+
+---
```
Si no tiene Js puedes borrarlo.
```suggestion
```

colmedev replied:
Done!!

Zyruks commented on src/components/Navbar/Navbar.astro:
```diff
@@ -0,0 +1,142 @@
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
+      d="M6.52155 8.91282C6.38265 8.79585 6.26568 8.65695 6.19258 8.49611C6.11216 8.19638 6.14871 7.90395 6.24375 7.61153C6.40458 7.14365 6.67508 6.73425 6.93095 6.31023C7.20876 5.85697 7.54504 5.45489 7.88133 5.06011C8.56122 4.25594 9.35077 3.56875 10.1842 2.92541C11.0907 2.2309 12.0484 1.63143 13.1523 1.30976C13.7445 1.1343 14.3512 1.0612 14.958 1.18548C16.1131 1.41942 16.8515 2.51601 16.6907 3 ...
```
Agregar TODO: de mover esto a un componente

colmedev replied:
Done

Zyruks commented on src/components/Navbar/Navbar.astro:
```diff
@@ -0,0 +1,142 @@
+      </li>
+      <li class="dots">&#183;</li>
+      <li>
+        <!-- Here it goes the toggle button for the lang -->
```
```suggestion
        <!-- TODO: Here it goes the toggle button for the lang -->
```

colmedev replied:
Done!

Zyruks commented on src/components/Navbar/Navbar.astro:
```diff
@@ -0,0 +1,142 @@
+  <!-- Navigation Section -->
+  <nav class="navbar">
+    <ul class="list">
+      <li class="listItem">
```
```suggestion
      <li class="list-item">
```

Intentar mantener las classes en lower case

colmedev replied:
Done!

Zyruks commented on src/components/Navbar/Navbar.astro:
```diff
@@ -0,0 +1,142 @@
+    display: none;
+  }
+
+  @media (width >= 50rem) {
```
Fuera buena tener media queries estandarizada, así luego no tener problemas con diferentes medias queries en toda la página.

Zyruks commented on src/pages/index.astro:
```diff
@@ -7,6 +8,7 @@ import Ripley from './_Sections/Ripley/Ripley.astro';
 
 <Layout title="Welcome to Astro.">
   <main>
+    <Navbar />
```
```suggestion
  <Navbar />
       <main>
  
```

En semantic el main header va fuera de main 

Es decir seria algo tal que 

```html
<header />
<main />
<footer />
```

colmedev replied:
Done

Zyruks reviewed (changes requested):
Pocas cositas que arreglar, buen PR.

KEEP GOING!

colmedev committed:
refactor(pages): :art: fixing the semantic

colmedev committed:
refactor(pages): :art: fixing the semantic

colmedev committed:
refactor(pages): :art: fixing the semantic

colmedev committed:
refactor(pages): :art: fixing the semantic

colmedev committed:
refactor(pages): :art: fixing the semantic

colmedev committed:
feat(components): :art: updating the mobile design

colmedev committed:
feat(components): :art: updating the mobile design

colmedev committed:
feat(components): :art: updating the mobile design

colmedev committed:
feat(components): :art: updating the mobile design

colmedev committed:
feat(components): :art: fixing the mobile menu

colmedev committed:
feat(components): :art: fixing the mobile menu

colmedev committed:
feat(components): :art: fixing the mobile menu

colmedev committed:
feat(components): :art: fixing the mobile menu

colmedev committed:
feat: the class now are in lowercase

Zyruks committed:
chore(pages): 🚚 update paths

colmedev committed:
Merge branch 'feature/navbar' of github.com:colmedev/anarangel.github.io into feature/navbar

colmedev committed:
feat(layouts): :sparkles: give animation to navbar

Zyruks committed:
refactor(header): ♻️ update navbar and SVG components with accessibility and style enhancements

- Update navbar ID for clearer semantics.
- Add 'width' attribute to SVG for direct HTML sizing.
- Replace TO DO comments with TODO for standardization.
- Implement data attributes for dynamic styling and accessibility in the navigation bar.
- Transition inline styles to SCSS for better maintainability and readability.
- Refactor JavaScript for navbar toggle to use data attributes for state management.

This refactor enhances code clarity, maintainability, and accessibility while ensuring
the style and behavior of the navbar component remain consistent with design expectations.

BREAKING CHANGE: The 'id' attribute of the navbar and the class names for the hamburger menu
bars are changed, which could affect external scripts or stylesheets directly referencing
the old IDs or classes.

colmedev committed:
feat: :fire: remove the navbar folder

colmedev committed:
feat: index.ts change cause removing the navbar folder

colmedev committed:
feat: :hammer: the class "hidden" now is more specific with the name "hide-navbar"

colmedev committed:
fix: :fire: removing navbar id and add class logo
